### DISCUSSION TITLE
Welcome to WinDynamicDesktop Discussions!

### DISCUSSION THREAD

**Discussion by t1m0thyj** [t1m0thyj/WinDynamicDesktop#279] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
I changed some of the translations in the app for Russian language. Now it sounds less like the language of machines

**Reply by t1m0thyj** [reply]
Thanks! Will include the translation updates soon in the next release of WDD.

**Comment by shubham3700x** [comment]
I WANNA UPLOAD MY ZIP FILE TO YOUR WEBSITE .
HOW CAN I DO THAT PLS HELP.
[ColourfulDAY.zip](https://github.com/t1m0thyj/WinDynamicDesktop/files/6503544/ColourfulDAY.zip)

**Reply by t1m0thyj** [reply]
Please follow the instructions here: https://github.com/t1m0thyj/WDD-themes#submit-new-themes

**Comment by goforit8** [comment]
### Taskbar Does Not Auto-Hide in Non-Maximized Windows Causing Obstruction in Windows 11
In Windows 11, there is an inconvenience with the taskbar not automatically hiding when an application window is in a restored state rather than maximized. This frequently results in the taskbar obstructing the workspace and interfering with user operations. It is hoped that this behavior can be addressed and improved in future updates.

**Reply by t1m0thyj** [reply]
I agree this is an annoying bug in Win11, but I don't think there is anything that WinDD can do about it. WinDD should have minimal impact to the taskbar since it runs in the background most of the time.
